pkg/capture/capture_manager.go

```diff
@@ -454,6 +458,35 @@ func (cm *Manager) rotate(ctx context.Context, writeoutChan chan<- capturetypes.
 	).Debug("rotated interfaces")
 }
 
+func (cm *Manager) logErrors(ctx context.Context, iface string, errsChan <-chan error) {
+	logger := logging.FromContext(ctx)
+	for {
+		select {
+		case <-ctx.Done():
+			return
+		case err, ok := <-errsChan:
+			if !ok {
+
+				// Ensure there is no conflict with calls to update() that might already be
+				// taking down this interface
+				cm.Lock()
```

Comment:
This should avoid any conflicts with other methods that may be closing the capture.